src/textures/DepthTexture.js

```diff
@@ -23,16 +23,19 @@ class DepthTexture extends Texture {
 	 * @param {number} [minFilter=LinearFilter] - The min filter value.
 	 * @param {number} [anisotropy=Texture.DEFAULT_ANISOTROPY] - The anisotropy value.
 	 * @param {number} [format=DepthFormat] - The texture format.
+	 * @param {number} [depth=1] - The depth of the texture.
 	 */
-	constructor( width, height, type = UnsignedIntType, mapping, wrapS, wrapT, magFilter = NearestFilter, minFilter = NearestFilter, anisotropy, format = DepthFormat ) {
+	constructor( width, height, type = UnsignedIntType, mapping, wrapS, wrapT, magFilter = NearestFilter, minFilter = NearestFilter, anisotropy, format = DepthFormat, depth = 1 ) {
```

Comment:
It's at times like these that I think that object-based function parameters are better, functions with more than 4 parameters start to get confusing for me.

Reply:
It would be possible to define something like the example below, Instead of an additional parameter?

```js
const depthMap = new THREE.DepthTexture( ... );
depthMap.depth = value; 
```

Reply:
If we go this route we would need to apply the `isArrayTexture` patch later in the code, for example in the `Textures` class or apply the same patch as I did in the constructor of `Texture`.

Reply:
I know we are dealing with limitations but the function signature is inappropriate. Until we adopt a convention we can make this more explicit perhaps even with `DepthArrayTexture` again. The class name and parameters also has the function of making the code easier to read.